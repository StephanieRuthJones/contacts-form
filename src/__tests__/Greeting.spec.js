
import React from "react"
import {render, cleanup} from "@testing-library/react"
import Greeting from "../components/Greeting"

afterEach(cleanup)

describe("Greeting", () => {  
  it("greets the user", () => {
    const {asFragment} = render(<Greeting name={"Qualified"} />)
    expect(asFragment()).toMatchSnapshot()
  })
  
  it("greets the user when the name is unknown", () => {
    const {asFragment} = render(<Greeting />)
    expect(asFragment()).toMatchSnapshot()
  })
})