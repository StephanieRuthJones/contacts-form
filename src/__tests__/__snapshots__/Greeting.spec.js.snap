// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting greets the user 1`] = `
<DocumentFragment>
  <div>
    Hello there!
  </div>
</DocumentFragment>
`;

exports[`Greeting greets the user when the name is unknown 1`] = `
<DocumentFragment>
  <div>
    Hello there!
  </div>
</DocumentFragment>
`;
